src/metabase/driver/mysql.clj

```diff
@@ -65,7 +65,8 @@
       kdb/mysql
       ;; 0000-00-00 dates are valid in MySQL, but JDBC barfs when queries return them because java.sql.Date doesn't allow it.
       ;; Add a param to the end of the connection string that tells MySQL to convert 0000-00-00 dates to NULL when returning them.
-      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull"))))
+      ;; Also add params to force UTF-8 encoding of results
+      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF8&characterSetResults=UTF8"))))
```

Comment:
Like if someone wants to filter by `name LIKE 'Cam Saül'`. That won't work if the connection is using a encoding incompatible with UTF-8 because that's what the frontend and backend code both assume.